src/cli.rs

```diff
@@ -394,14 +459,17 @@ impl Cli {
         let matches = app.try_get_matches_from_mut(iter)?;
         let mut cli = Self::from_arg_matches(&matches)?;
 
+        #[allow(clippy::single_match)]
         match cli.raw_method_or_url.as_str() {
             "help" => {
                 app.print_long_help().unwrap();
                 println!();
                 safe_exit();
             }
-            "print_completions" => return Err(print_completions(app, cli.raw_rest_args)),
-            "generate_completions" => return Err(generate_completions(app, cli.raw_rest_args)),
+            #[cfg(feature = "man-completion-gen")]
+            "generate-completions" => return Err(generate_completions(app, cli.raw_rest_args)),
+            #[cfg(feature = "man-completion-gen")]
+            "generate-manpages" => return Err(generate_manpages(app, cli.raw_rest_args)),
```

Comment:
If the feature isn't enabled it would be better to error out instead of treating these as hostnames.

Reply:
I've changed these subcommands to panic when the `man-completion-gen` feature is not enabled

Reply:
I think an error is more appropriate because it's expected to be possible to trigger it

Reply:
Now, it returns an error

```
$ cargo run -q -- generate-manpages doc
error: generate-manpages requires enabling man-completion-gen feature
```